src/test/java/api/SiteApi.java

```diff
@@ -0,0 +1,28 @@
+package api;
+
+
+import com.jayway.restassured.RestAssured;
+
+
+public class SiteApi extends RestAssured {
+    public SiteApi() {
+        this("https://www.treatwell.co.uk", "api");
+    }
+
+    public SiteApi(String baseURI, String basePath) {
+        RestAssured.baseURI = baseURI;
+        RestAssured.basePath = basePath;
+    }
+
+
```

Comment:
Hmm, looks like you've got two lines spacing between these functions. The Java standard is to have only one.